gcd/src/test/java/org/spine3/server/storage/datastore/DsEntityStorageShould.java

```diff
@@ -38,11 +41,6 @@
 
```

Comment:
This base class is generified no, please use generic params. And also check other test classes, they can be generified, too.